### DISCUSSION TITLE
missing wild card for neg prompt, but is inside file

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#3127] [Q&A]
getting this error for a few days

[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg

inside config file

    "path_wildcards": "/Users/Quad-Core/Fooocus/wildcards",
    "path_outputs": "/Volumes/1TB Caddy bay 5",
    "default_prompt_negative": "__neg__",
    "default_refiner": "",
    "default_prompt": "",
    "default_loras_min_weight": -8,
    "default_loras_max_weight": 8,  
    "default_advanced_checkbox": true,
    "default_max_image_number": 12,

inside wild card folder 100% in there

<img width="2560" alt="Screenshot 2024-06-09 at 09 02 18" src="https://github.com/lllyasviel/Fooocus/assets/35565479/cbd45225-d609-42d0-9754-5de775ab7a7f">

and again error on adding a image prompt/Users/Quad-Core/Fooocus/modules/core.py:257: RuntimeWarning: invalid value encountered in cast

black screen on preview

**Comment by mashb1t** [comment]
I assume you've already clicked on the reload files button and restarted Fooocus, correct?

**Comment by lgwjames** [comment]
yep, just loaded today as just go home

[Fooocus Model Management] Moving model(s) has taken 2.43 seconds
[Fooocus] Processing prompts ...
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Wildcards] processing: __neg__
[Wildcards] Warning: neg.txt missing or empty. Using "neg" as a normal word.
[Wildcards] neg
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] wedding ceremony - bride an groom facing each other in front of a wooden arbour, solemn, elegant, cinematic, focus, highly detailed, dramatic light, beautiful, intricate, innocent, expressive, perfect, cheerful, unique, romantic, inspired, colorful, very inspirational, designed, tender, vibrant, color, ambient background, epic, detail, focused, professional
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] wedding ceremony - bride an groom facing each other in front of a wooden arbour, decorated bright, cinematic, directed, extremely highly detailed, beautiful, intricate, perfect light, focus, fair, pristine, magnificent, professional, winning, real, full color, elegant, epic, sharp, artistic, vibrant colors, surreal, iconic, fine, aesthetic, very inspirational
[Fooocus] Preparing Fooocus text #3 ...
[Prompt Expansion] wedding ceremony - bride an groom facing each other in front of a wooden arbour, highly detailed, bright colors, cinematic, sharp focus, surreal, still, full color, perfect intricate, elegant, dynamic dramatic background, professional composition, beautiful, extremely aesthetic, stunning, creative, cheerful, amazing, symmetry, cool, awesome, illuminated, pretty, attractive, great
[Fooocus] Preparing Fooocus text #4 ...
[Prompt Expansion] wedding ceremony - bride an groom facing each other in front of a wooden arbour, solemn, elegant, highly detailed, sharp focus, perfect composition, cinematic, joyful, delicate, intricate, extremely artistic, colorful, surreal, glorious, light, illuminated, beautiful, majestic, dramatic, complex, very inspirational, amazing, thought, epic, colossal, cool, enhanced
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding positive #3 ...
[Fooocus] Encoding positive #4 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Encoding negative #3 ...
[Fooocus] Encoding negative #4 ...
[Fooocus] Image processing ...
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
Requested to load Resampler
Loading 1 new model
Requested to load To_KV
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.14 seconds
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 25.07 seconds
[Fooocus] Preparing task 1/4 ...
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 9.99 seconds
  0%|                                                                                                                                                        | 0/60 [00:00<?, ?it/s]huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
/Users/Quad-Core/Fooocus/modules/anisotropic.py:132: UserWarning: The operator 'aten::std_mean.correction' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/mps/MPSFallback.mm:13.)
  s, m = torch.std_mean(g, dim=(1, 2, 3), keepdim=True)
/Users/Quad-Core/Fooocus/modules/core.py:257: RuntimeWarning: invalid value encountered in cast
  x_sample = x_sample.cpu().numpy().clip(0, 255).astype(np.uint8)
  5%|███████▏                                                                          


black screen again!!! error again

**Reply by mashb1t** [reply]
Sorry, sadly i can't reproduce the black image issue on any of my systems, maybe somebody else can support you here.

**Comment by Rogala** [comment]
I got the same error when I converted the file to UTF-16 encoding, while UTF-8 is needed
![image](https://github.com/lllyasviel/Fooocus/assets/7437582/323439ad-b00a-4f44-97f5-52c448d9713c)

**Reply by mashb1t** [reply]
@Rogala thanks for the insights.
@lgwjames please check which encoding you're using, thanks!
